services/apps/alcs/src/alcs/application-decision/application-decision-v2/application-decision/application-decision-v2.service.spec.ts

```diff
@@ -553,4 +566,34 @@ describe('ApplicationDecisionV2Service', () => {
       );
     });
   });
+
+  // it('should throw an exception on release decision if no components', async () => {
```

Comment:
Commented out test?

Reply:
thanks! it is not relevant to this service and I forgot to delete it, I'll do it now